topic_folders/fundraising/collaborating-on-grants.md

```diff
@@ -1,22 +1,22 @@
 # A Guide for Collaborating with The Carpentries on Grants
 
 ## Statement to Interested Collaborators
-This guide was developed for potential collaborators interested in involving The Carpentries in a grant proposal. We value all contributions to our mission and are excited to collaborate with individuals and organisations interested in mission-aligned activities (i.e. Carpentries workshops, instructor training, curriculum development). We offer two options for collaborating with The Carpentries on grant proposals:
+This guide was developed for potential collaborators interested in involving The Carpentries in a grant proposal. We value all contributions to our mission and are excited to collaborate with individuals and organisations interested in mission-aligned activities (i.e., Carpentries workshops, instructor training, curriculum development). We offer two options for collaborating with The Carpentries on grant proposals:
 
-1. **Request a letter of support**. If you are planning to budget Carpentries programming (i.e. workshops, membership, instructor training seats) into your grant proposal and would like to request a letter of support, please [complete this form](https://carpentries.typeform.com/to/oK25vSPE). Pricing for Carpentries programming is located on our [membership page](https://carpentries.org/membership/). All requests for letters of support require **five business days** to be fulfilled.
+1. **Request a letter of support**. If you are planning to budget Carpentries programming (i.e., workshops, membership, instructor training seats) into your grant proposal and would like to request a letter of support, please [complete this form](https://carpentries.typeform.com/to/oK25vSPE). Pricing for Carpentries programming is on our [membership page](https://carpentries.org/membership/). All requests for letters of support require **five business days** to be fulfilled.
```

Comment:
Our style guide states to not include a comma in this case: https://docs.carpentries.org/topic_folders/communications/resources/style-guide.html#i-e